lib/client/hmr.js

```diff
@@ -10,7 +10,7 @@
 */
 const { error, info, refresh, warn } = require('./log');
 
-const { apply, check, status } = module.hot;
+const { apply, check, status } = module.hot || {};
```

Comment:
since these methods are only used in the `replace` method, we should move this assignment into that function.